packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -301,16 +302,16 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
 };
 
 const participantStateStringTrampoline = (
-  props: ParticipantItemProps,
-  strings: ParticipantItemStrings
+  strings: ParticipantItemStrings,
+  participantState?: ParticipantState
 ): string | undefined => {
   /* @conditional-compile-remove(one-to-n-calling) */
   /* @conditional-compile-remove(PSTN-calls) */
-  return props.participantState === 'Idle' || props.participantState === 'Connecting'
```

Comment:
Why are we only updating this only for idle and not connecting as well?
A Teams user joining a call has the connecting state:
![image](https://user-images.githubusercontent.com/2684369/211422881-a0d45c11-382a-49e7-8777-732553b4f7d9.png)
(https://skype.visualstudio.com/SPOOL/_workitems/edit/3075499)

It's confusing that participantStateConnecting string is "Calling...":
![image](https://user-images.githubusercontent.com/2684369/211423016-9b5c652b-3325-448e-a8dc-47b878ce46c1.png)

Reply:
Good point! We should actually take a look at the userId type here. 'Connecting' is only important for PSTN users. will take another look at this. fix will address the below comment as well.

Reply:
This was updated from the changes in the selectors that now filter out all remote participants in the idle state,